benchmark/main.cpp

```diff
@@ -961,8 +964,14 @@ static int benchmark( int argc, char *argv[] )
 }
 
 int main(void) {
-    int ret = benchmark(0, NULL);
+    mbedtls_platform_context platform_ctx;
+    if(mbedtls_platform_setup(&platform_ctx) != 0) {
+        return -1;
+    }
+
+    int ret = benchmark(0, NULL, &platform_ctx);
     if (ret != 0) {
         mbedtls_printf("Benchmark failed with error %d\r\n", ret);
     }
+    mbedtls_platform_teardown(&platform_ctx);
```

Comment:
Note that there are a lot of places inside `benchmark()` that call the function `mbedtls_exit()`. In these cases, the `mbedtls_platform_teardown()` will not be executed. Is this the intended behaviour?